Frontend-v1-Original/components/layout/layout.tsx

```diff
@@ -45,7 +70,28 @@ export default function Layout({
           </>
         )}
         <SnackbarController />
-        <main>{children}</main>
+        {!chain?.unsupported ? (
+          <main>{children}</main>
+        ) : (
+          <div className="flex flex-grow items-center justify-center text-center">
+            <div>
```

Comment:
```suggestion
            <div className="space-y-2">
```
Might give the text and button a little space.